<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_root</class>
 <widget class="QWidget" name="w_root">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cppexec</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="split_input_control">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="split_io_vert">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="split_io_horiz">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="w_input" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="l_input">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Input:</string>
           </property>
           <property name="buddy">
            <cstring>txt_input</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="btn_maximize">
           <property name="text">
            <string>Ma&amp;ximize</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="txt_input">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
            </font>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="tabStopWidth">
            <number>40</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="w_runtime" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Runtime Output:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Exit Code:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="line_run_exit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>131</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTextEdit" name="txt_run_output">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="w_output" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="l_output">
          <property name="text">
           <string>Compiler Output:</string>
          </property>
          <property name="buddy">
           <cstring>txt_compile_output</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>243</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="l_exit_code">
          <property name="text">
           <string>Exit Code:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="line_compile_exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTextEdit" name="txt_compile_output">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Mono'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="tabStopWidth">
           <number>40</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="w_controls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <widget class="QWidget" name="w_command" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="0">
           <widget class="QPushButton" name="btn_crear">
            <property name="text">
             <string>&amp;Clear</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_compile_run">
            <property name="text">
             <string>Compile and &amp;Run</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="w_path" native="true">
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Li&amp;brary Path (-L):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>line_library_path</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLineEdit" name="line_library"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Library (-l):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>line_library</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="line_include_path"/>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="line_library_path"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>I&amp;nclude Path (-i):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>line_include_path</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="w_options" native="true">
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Options:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="buddy">
             <cstring>opt_wall</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="opt_wall">
            <property name="text">
             <string>-Wall (compiler warnings)</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="w_target_lang" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Target:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="buddy">
             <cstring>c99</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="c99">
            <property name="text">
             <string>C99</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="cpp98">
            <property name="text">
             <string>C++98</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_input</tabstop>
  <tabstop>btn_maximize</tabstop>
  <tabstop>btn_compile_run</tabstop>
  <tabstop>btn_crear</tabstop>
  <tabstop>cpp98</tabstop>
  <tabstop>c99</tabstop>
  <tabstop>opt_wall</tabstop>
  <tabstop>line_include_path</tabstop>
  <tabstop>line_library_path</tabstop>
  <tabstop>line_library</tabstop>
  <tabstop>txt_compile_output</tabstop>
  <tabstop>line_compile_exit</tabstop>
  <tabstop>line_run_exit</tabstop>
  <tabstop>txt_run_output</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>txt_compile_output</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>916</x>
     <y>654</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>line_compile_exit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>txt_run_output</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>980</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>line_run_exit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>954</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>txt_input</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>969</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>line_library_path</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>990</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>993</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>line_library</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1007</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>936</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_maximize</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_runtime</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_maximize</sender>
   <signal>toggled(bool)</signal>
   <receiver>w_output</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>671</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_crear</sender>
   <signal>clicked()</signal>
   <receiver>line_include_path</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>894</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>891</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
